Tools/cases_generator/analyzer.py

```diff
@@ -1079,6 +1081,36 @@ def add_instruction(name: str) -> None:
     return instmap, len(no_arg), min_instrumented
 
 
+def is_instruction_size_macro(token: lexer.Token) -> bool:
+    return token.kind == "IDENTIFIER" and token.text == "INSTRUCTION_SIZE"
```

Comment:
```suggestion
    return token.text == "INSTRUCTION_SIZE"
```
If the text is "INSTRUCTION_SIZE" then the token must be an identifier.